src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutesV1Spec.scala

```diff
@@ -11,10 +12,14 @@ import org.broadinstitute.dsde.workbench.sam.model._
 import org.broadinstitute.dsde.workbench.sam.service._
 import org.scalatest.concurrent.ScalaFutures
 
+import scala.concurrent.duration._
+
 /**
   * Unit tests of GoogleExtensionRoutes. Can use real Google services. Must mock everything else.
   */
 class GoogleExtensionRoutesV1Spec extends GoogleExtensionRoutesSpecHelper with ScalaFutures{
+  implicit val timeout = RouteTestTimeout(5 seconds) //after using com.google.cloud.storage.contrib.nio.testing.LocalStorageHelper, tests seems to run a bit longer
```

Comment:
If this is required to account for the 2 second sleep (above) then maybe worth adding a similar comment here about why we're adding this in and when we can remove it. It seems like this would change the timeout for all tests in this suite, which might not be something we want to keep long term?

Reply:
This timeout already existed in `GoogleExtensionRoutesSpec` with the same explanatory comment seen here. Since that test suite is basically the same as this one, I don't think we'd remove this timeout even after we removed the 2s sleep. I'm guessing that this was just missed or wasn't showing the flakiness for some reason